src/ios/RNCSlider.h

```diff
@@ -23,6 +23,8 @@
 @property (nonatomic, strong) UIImage *maximumTrackImage;
 @property (nonatomic, strong) UIImage *thumbImage;
 @property (nonatomic, assign) bool tapToSeek;
+@property (nonatomic, assign) bool minimumTrackImageSet;
+@property (nonatomic, assign) bool maximumTrackImageSet;
```

Comment:
I don't think we should make these public - let's keep them private.

Reply:
Done.